Userland/Libraries/LibTLS/TLSv12.h

```diff
@@ -360,7 +360,7 @@ class TLSv12 final : public Core::Socket {
     static ErrorOr<NonnullOwnPtr<TLSv12>> connect(ByteString const& host, Core::Socket& underlying_stream, Options = {});
 
     using StreamVariantType = Variant<OwnPtr<Core::Socket>, Core::Socket*>;
-    explicit TLSv12(StreamVariantType, Options);
+    explicit TLSv12(StreamVariantType, Options, PreventSIGPIPE = PreventSIGPIPE::Yes);
```

Comment:
I'm wondering about makign this change to LibCore. I would, but I don't quite understand why we would ever set the flag to Sigpipe by default to begin with

Reply:
Yeah, if anyone ever wants to actually enable the behaviour, they can make a good argument and add the option later.